package.json

```diff
@@ -31,21 +31,9 @@
       "remote-storage-mappings": "1.0"
     },
     "permissionSets": [
-      {
-        "permissionName": "module.tenant-settings.enabled",
-        "displayName": "UI: Tenant-settings module is enabled",
-        "visible": false
-      },
```

Comment:
We still need this one around; [`stripes-core` enforces `module.foo.enabled` and `settings.foo.enabled`](https://github.com/folio-org/stripes-core/blob/master/doc/permissions.md#which-permissions-defined-where).

Reply:
good catch @zburke!

https://github.com/folio-org/stripes-core/blob/6fd524c42a8894fc460cfd25f7b92428d70fecaf/src/moduleRoutes.js#L26